src/main/resources/application.properties

```diff
@@ -0,0 +1,12 @@
+# Database
+spring.datasource.driver-class-name=com.mysql.jdbc.Driver
+spring.datasource.url=jdbc:mysql://localhost:3306/onlinestore?serverTimezone=Europe/Minsk&useSSL=false
+spring.datasource.username=root
+spring.datasource.password=1111
+spring.jpa.hibernate.ddl-auto=none
+spring.jpa.database-platform=org.hibernate.dialect.MySQL5Dialect
```

Comment:
Use _MySQL57InnoDBDialect _ to manage relations, default _MySQL5Dialect_ doesn't support it